Sources/Client/Model/Channel/Channel.swift

```diff
@@ -122,69 +118,33 @@ public final class Channel: Codable {
     /// An event when the channel was updated.
     public var onUpdate: OnUpdate<Channel>?
     
-    let needsCreation: Bool
-    
-    /// Init a channel 1-by-1 (direct message) with another member.
-    /// - Parameters:
-    ///   - type: a channel type.
-    ///   - member: an another member.
-    ///   - extraData: an `Codable` object with extra data of the channel.
-    ///   - currentUser: a current user, should be `Client.shared.user`.
-    public convenience init(type: ChannelType,
-                            with member: Member,
-                            extraData: Codable? = nil,
-                            currentUser: User = Client.shared.user) {
-        var members = [member]
-        
-        if member.user != currentUser {
-            members.append(currentUser.asMember)
-        }
-        
-        self.init(type: type,
-                  id: "",
-                  name: member.user.name,
-                  imageURL: member.user.avatarURL,
-                  members: members,
-                  extraData: extraData)
-    }
+    /// Checks if the channel was decoded from a channel response.
+    public let didLoad: Bool
```

Comment:
Can we improve naming of this property? Maybe `decoded` or `isDecoded`?

Reply:
I don't like this flag at all. I really want to avoid it.
I think `didLoad` is more similar to developers.
`decoded` and `isDecoded` like a hardcoded thing.